### PR TITLE
Develop property model and offer model

### PR DESCRIPTION
**Pull request by storres21**
Code created to connect the app to the real-time database, apart from that I created the two services, one to create a restriction in the type of the offer and other to upload images form firebase storage

### CODE REVIEW

File: Andlet/Andlet/Views/Offers/OfferView.swift [hunk 1]
```diff
@@ -48,6 +48,12 @@ struct OfferView: View {
                     Text("\(1) 🛁")
                         .font(.custom("LeagueSpartan-SemiBold", size: 16))
                         .foregroundColor(Color(hex: "000000"))
+                    Text("|")
+                        .font(.custom("LeagueSpartan-SemiBold", size: 16))
+                            .foregroundColor(Color(hex: "000000"))
+                    Text("\(3) 🧑‍🤝‍🧑")
```
Comment (Daniel7578) [inline Andlet/Andlet/Views/Offers/OfferView.swift:54]: Does this Icon will not provoke any crashes in the app?

File: Andlet/Andlet/Utils/StorageService.swift
Comment (Daniel7578) [inline Andlet/Andlet/Utils/StorageService.swift:1]: Does this service work across the hall application?

File: Andlet/Andlet/Utils/OfferTypeService.swift
Comment (Daniel7578) [inline Andlet/Andlet/Utils/OfferTypeService.swift:1]: This logic should be handled by the viewModel ?

Comment (Daniel7578) [reply Andlet/Andlet/Utils/OfferTypeService.swift:1]: Or the viewModel will use this service in the future ?

File: Andlet/Andlet/Views/Offers/OfferView.swift (hunk 1 above)
Comment (storres21) [reply Andlet/Andlet/Views/Offers/OfferView.swift:54]: No!! This icon is used to include the smart feature in the app

File: Andlet/Andlet/Utils/StorageService.swift
Comment (storres21) [reply Andlet/Andlet/Utils/StorageService.swift:1]: Yes!! But it is mainly made to use in the process of creating a new listing

File: Andlet/Andlet/Utils/OfferTypeService.swift
Comment (storres21) [reply Andlet/Andlet/Utils/OfferTypeService.swift:1]: Correct, the viewModel will use that service in the future

**Review by Daniel7578 — APPROVED** [review]

**Pull request merged**